spec/models/gallery_spec.rb

```diff
@@ -1,55 +1,7 @@
 # frozen_string_literal: true
 require 'models/concerns/is_permissionable_examples'
 RSpec.describe Gallery do
```

Comment:
We should have tests to ensure that `Gallery#set_images_from_portal_urls` results in `GalleryImage` records with `url` attribute set from `edmIsShownBy`.

Reply:
tests added here: 7d0d1f27c9ba19c3992065c2a02d23e866bc4e91